#5674. Add documentation to configure local environment with Tomcat (#5676)
geosolutions-it/MapStore2: docs/developer-guide/developing.md

* You have to run npm start to run mapstore client on port 8081, that is now connected to the local test back-end

> ### Using local backend and setting up tomcat server

Reviewer: ```suggestion
### Running local backend in tomcat 
```

> * Download a tomcat standalone [here](https://tomcat.apache.org/download-90.cgi) and extract to a folder of your choice

Reviewer 2: We point to Tomcat 7 here: https://mapstore.readthedocs.io/en/latest/developer-guide/requirements/

Maybe it's better to be consistent and always suggest the same version, or just link the above page for tomcat installation and deploy.

Author: Yes I am making the changes.

Author: @mbarto made the changes

> * Download a tomcat standalone [here](https://mapstore.readthedocs.io/en/latest/developer-guide/requirements/) and extract to a folder of your choice

Reviewer: ```suggestion
If you prefer, or if you have some problems with `mvn jetty:run-war`, you can run MapStore back-end in a tomcat instance instead of using Jetty. 
To do so, you can :
* download a tomcat standalone [here](https://mapstore.readthedocs.io/en/latest/developer-guide/requirements/) and extract to a folder of your choice
```

* To generate a war file that will be deployed on your tomcat server, go to the root of the Mapstore project that was git cloned and run `./build.sh`. This might take some time but at the end a war file named `mapstore.war` will be generated into the `web/target` folder.
* Copy the `mapstore.war` and then head back to your tomcat folder. Look for a `webapps` folder and paste the `mapstore.war` file there.

> * For Unix systems, to start tomcat server, go to the terminal `cd` into the root of your tomcat extracted folder and run `./bin/startup.sh`
> * The server will start on port `8080` and Mapstore will be running at `http://localhost:8080/mapstore`
> * But for development purposes, we're interested in the backend that was started on the tomcat server along with Mapstore.

Reviewer: ```suggestion
* To start tomcat server, go to the terminal, `cd` into the root of your tomcat extracted folder and run `./bin/startup.sh` ( unix systems) or `./bin/startup.bat` (Windows). The server will start on port `8080` and Mapstore will be running at `http://localhost:8080/mapstore`. For development purposes we're only interested in the backend that was started on the tomcat server along with Mapstore.
```
